pkg/server/ca/upstream_client_test.go

```diff
@@ -31,7 +31,9 @@ func TestUpstreamClientMintX509CA_HandlesBundleUpdates(t *testing.T) {
 		UseIntermediate: true,
 	})
 
-	x509CA, err := client.MintX509CA(context.Background(), csr, 0)
+	x509CA, err := client.MintX509CA(context.Background(), csr, 0, func(_ context.Context, _, _ []*x509.Certificate) error {
```

Comment:
maybe we can add a negative case where validation fails?

Reply:
The manager tests have a test which exercises it, but I can add an explicit one here.